src/test/java/com/coutemeier/maven/plugins/asciidoctor/lifecycle/PrepareSourcesIT.java

```diff
@@ -0,0 +1,22 @@
+package com.coutemeier.maven.plugins.asciidoctor.lifecycle;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import io.takari.maven.testing.executor.MavenRuntime.MavenRuntimeBuilder;
+
+public class PrepareSourcesIT
+extends AbstractMojoIT {
+    public PrepareSourcesIT( final MavenRuntimeBuilder builder )
+    throws Exception {
+        super( builder );
+    }
+
+    @Test
+    public void prepareSourcesIOExceptionTest()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/asciidoctor-lifecycle-maven-plugin/asciidoctor-lifecycle-maven-plugin/pullRequest?prid=5396310)